### ISSUE TITLE
Image Processing in node 

### ISSUE THREAD

**Issue by bastianovish** [BuildingXwithJS/proposals#11]
I Would like to see some Compute Vision in node, like Face / Object detection, Snapchat-like filters, or something like that!. There is a ton of examples in other languages like Python, but not so much in node. Btw thanks for your videos!

**Comment by yamalight** [comment]
Yep, wanted to do opencv with node for some time. Thanks for reminding me 👍

**Comment by FDiskas** [comment]
https://cloudinary.com/ have some nice integrations and use cases
